### DISCUSSION TITLE
Missing pagination information from `/v2/playlists/me`

### DISCUSSION THREAD

**Discussion by 1729patrick** [tidal-music/discussions#97] [Report bug]
Hey guys,

Thank you for providing the API and for the exciting upcoming features!

I've created several playlists for my user and added one song to each playlist to test the pagination for `/v2/playlists/me`. While the API returns the first 20 results, I'm not seeing the `links.next` string to fetch the next page, even though I have more than 20 playlists.

<img width="630" alt="image" src="https://github.com/user-attachments/assets/27eebda4-e521-4375-963a-8ccc2ccb19ee">

```
curl --request GET \
  --url https://openapi.tidal.com/v2/playlists/me \
  --header 'Authorization: Bearer eyJraWQiOiJ2OU1GbFhqWSIsImFsZyI6IkVTMjU2In0.eyJ0eXBlIjoibzJfYWNjZXNzIiwidWlkIjoyMDA2NjQyODYsInNjb3BlIjoicGxheWxpc3RzLnJlYWQiLCJnVmVyIjowLCJzVmVyIjowLCJjaWQiOjEzODU2LCJleHAiOjE3MjY1NzA3NDMsInNpZCI6ImU3ZjVlNWE5LWRhYWYtNGUzYi1iNzcxLTViYWU2YTk3NmRmMCIsImlzcyI6Imh0dHBzOi8vYXV0aC50aWRhbC5jb20vdjEifQ.veXLUEghSJv-w1M_Av2jpunMq6eaaEoAz3UmWfTXJELX805zprflOcZOxwdaUwJuV5t1KeurT0q4w71p9e0dPQ'
```

**Comment by aliaksei-taliuk** [comment]
@1729patrick hi, thanks for pointing this out. I will check this out and get back to you shortly.

**Reply by aliaksei-taliuk** [reply]
@1729patrick fixed, please free to reach out if you spot anything else. Thanks!

**Comment by aliaksei-taliuk** [comment]
Assumed, it's fixed now.
